### ISSUE TITLE
support flake8-fixme

### ISSUE THREAD

**Issue by finswimmer** [charliermarsh/ruff#3859]
Hey,

until know I could find rules for all my flake8 plugins or at least a feature request to support those.

The only one I'm missing is [flake8-fixme](https://pypi.org/project/flake8-fixme/). 

Would be great to see this supported :)

fin swimmer

**Comment by edgarrmondragon** [comment]
Maybe https://github.com/orsinium-labs/flake8-todos would be preferred?

It seems more comprehensive

<blockquote>

+ **T001**: use TODO instead of FIXME (or BUG) for consistency.
+ **T002**: add author into TODO ([motivation](https://dave.cheney.net/practical-go/presentations/qcon-china.html#_dont_comment_bad_code_rewrite_it)).
+ **T003**: add link on issue into TODO.
+ **T004**: missed colon in TODO.
+ **T005**: missed text in TODO.
+ **T006**: write TODO instead of ToDo (use upper case).
+ **T007**: missed space after colon in TODO.

</blockquote>

**Comment by finswimmer** [comment]
> Maybe https://github.com/orsinium-labs/flake8-todos would be preferred?

The purpose if `flake8-todos` seems to another one, compared to `flake8-fixme`. The later one ensures that there are not `TODO`s, `FIXME` etc. in the code base. And that's the functionality I like to have in `ruff` :smiley:

**Comment by edgarrmondragon** [comment]
You're right, the two plugins address different problems! I'll create a separate issue for my suggestion.

**Comment by evanrittenhouse** [comment]
@charliermarsh can you please assign this to me? Once #3921 lands, I'll be able to refactor some of the `TODO` detection out to a common location - when _that's_ done, it'll be pretty trivial to clear this as well.

**Comment by evanrittenhouse** [comment]
The fix I have in mind depends on [this PR](https://github.com/charliermarsh/ruff/pull/4558) being merged so I can rebase on top of it
